Sources/TuistKit/Commands/TestCommand.swift

```diff
@@ -46,6 +46,11 @@ struct TestCommand: ParsableCommand {
     )
     var configuration: String?
 
+    @Flag(
```

Comment:
Nitpick: I'd extend the annotation to be more explicit about the short version of the flag. Since it's UI tests, what about making it `-u`?

Reply:
Hmm, if at some point we decided to add `skip-unit-tests` option, wouldn't a `-u` short version be confusing? 🤔 Maybe `-ui`?